lib/stores/accounts_store.dart

```diff
@@ -163,6 +165,16 @@ class AccountsStore extends ChangeNotifier {
     ];
   }
 
+  Future<void> checkNotifications(UserData? userData) async {
+    notificationCount = await LemmyApiV3(userData!.instanceHost)
+        .run(GetUnreadCount(
+          auth: userData.jwt.raw,
+        ))
+        .then((e) => e.mentions + e.replies + e.privateMessages);
+
+    notifyListeners();
```

Comment:
All the others have a `return save();`? 
Do you need it here?

Reply:
No cause I didnt care about serializing notification count in prefs